lib/presentation/crowd_action/crowd_action_details.dart

```diff
@@ -0,0 +1,60 @@
+// import 'package:collaction_app/domain/crowdaction/crowdaction.dart';
```

Comment:
When commenting out classes that will be re-implemented, I  would state this in a TODO in the first line.